addon/ng2/models/webpack-build-utils.ts

```diff
@@ -19,5 +19,6 @@ export const webpackDevServerOutputOptions = {
   hash: true,
   timings: true,
   chunks: false,
-  chunkModules: false
+  chunkModules: false,
+  warnings: false
```

Comment:
Isn't  that going to remove _all_ warnings? Is there a way to specifically remove source-maps warnings?